### ISSUE TITLE
Classic Explorer status bar is light gray on Windows 10 1809 dark mode

### ISSUE THREAD

**Issue by amartini** [Open-Shell/Open-Shell-Menu#121]
I'm using dark mode and everything seems right except for the status bar. 

![screenshot 14](https://user-images.githubusercontent.com/894369/50068852-a7a42300-01ae-11e9-8ee1-1ca0ad584d41.png)

**Comment by blackcrack** [comment]
go in the Configuration of Open-shell menu, All Programms Skin => select smoked Glass, 
this match so far or custom it more with custom coloring the menu.

imho
 can be closed this issue 
Because the Open-Shell Menu it is possible to configuring with custom colors ..
isn't it @ge0rdi ?! 

best regards
Blacky

**Comment by amartini** [comment]
> go in the Configuration of Open-shell menu, All Programms Skin => select smoked Glass,
> this match so far or custom it more with custom coloring the menu.
> 
> imho
> can be closed this issue
> Because the Open-Shell Menu it is possible to configuring with custom colors ..
> isn't it @ge0rdi ?!
> 
> best regards
> Blacky

Haven't changed for me. besides i use windows 7 start menu, and smoked glass isn't available

**Comment by teroteki** [comment]
I too have a light gray Status Bar after upgrading to the new Win10 1809 dark theme.
I only install the "Classic Explorer" enhancements of Open-Shell (it's all I need as I use TrueLaunchBar/StartKiller etc. to handle the Start menu). 

Am I still able to select this "Smoked Glass" skin/theme somewhere via .ini editing?
Or do I have to install the "Open-Shell Menu" component just to get those settings? Actually, why does the "Open-Shell Menu" theme settings affect the "Classic Explorer" look?

**Comment by blackcrack** [comment]
on top => Show all settings
![screen 00000](https://user-images.githubusercontent.com/7925717/53384423-25d53080-397b-11e9-9e0b-85047b1b88b8.jpg)
![screen 00002](https://user-images.githubusercontent.com/7925717/53384490-5cab4680-397b-11e9-8cbf-510a64447f02.jpg)
![screen 00003](https://user-images.githubusercontent.com/7925717/53384555-98dea700-397b-11e9-8972-d20cc9ff8f1a.jpg)

**Comment by ghosted17** [comment]
I think there should just be an option for for explorer. Dark / white. And make the dark Match the color of dark explorer. You don't want it linked to the shell theme.

**Comment by amartini** [comment]
> on top => Show all settings

This have nothing to do with my initial claim. I'm talking about the status bar on windows explorer windows, as you can see on the screenshot.

**Comment by ghosted17** [comment]
Didn't change anythinghing for me either the status bar remains white and even if it did work I don't want to use classic theme just for that.

**Comment by NomDeMorte** [comment]
![image](https://user-images.githubusercontent.com/45681208/53490052-d01d9700-3ae6-11e9-87ac-102d9d14fb5a.png)
WFM.

**Comment by NomDeMorte** [comment]
This is a known bug in the dark mode of Explorer. This is a Microsoft bug, not an Open-Shell bug.

**Comment by ge0rdi** [comment]
> This is a known bug in the dark mode of Explorer. This is a Microsoft bug, not an Open-Shell bug.

No. Status bar (that second one) is added/drawn by `ClassicExplorer`.
We will need to fix `ClassicExplorer` so that it will use actual theme colors (for both status bar and toolbar).

Though for now I have no clue how to do it :(

**Comment by ghosted17** [comment]
Is the problem getting it to follow the skin color? Or can we not even compile a static dark version of the status bar?

**Comment by amartini** [comment]
Guys, i'm talking about the Status Bar on the Classic Explorer. This have nothing to do with task bar or start menu! The status bar is enabled with this setting:
![screenshot 47](https://user-images.githubusercontent.com/894369/53507208-3a5a1b80-3a96-11e9-9fd2-42ce9e0e49fb.png)

**Comment by ghosted17** [comment]
I think there saying that their trying to link that statusbar color in explorer to the applied theme in start menu. Correct me if I'm wrong?

**Comment by amartini** [comment]
Well maybe it's indeed a windows bug. I see the status bar is created on ExplorerBHO.cpp line 1256:

`m_Status8=CreateWindow(STATUSCLASSNAME,NULL,WS_CHILD|WS_VISIBLE|SBARS_SIZEGRIP|SBARS_TOOLTIPS,0,0,rc.right-rc.left,rc.bottom-rc.top,GetParent(status),NULL,g_Instance,NULL);`

I don't see it painting the background anywhere, so maybe `STATUSCLASSNAME` is broken.

**Comment by Hadden89** [comment]
Same on classic explorer toolbar. Still gray. Guess is related to this explorer bug.
(Open Shell menu works fine.)

**Comment by AkiraJkr** [comment]
Can confirm this is a issue. Dark mode selected in Windows 10 and the Classic Explorer bars don't match Dark Mode.

**Comment by ghosted17** [comment]
Still hoping for a fix

**Comment by individuell** [comment]
any fix for this?

**Comment by NomDeMorte** [comment]
> > This is a known bug in the dark mode of Explorer. This is a Microsoft bug, not an Open-Shell bug.
> 
> No. Status bar (that second one) is added/drawn by `ClassicExplorer`.

Ah I sse now. This was an MS bug, so I was just accustomed to seeing a light status bar there, but MS's one is now fixed.

**Comment by Ibuprophen** [comment]
I'm working on getting the latest Windows 10 reinstalled on one of my Laptops again. 

I was doing some playing around with a few things (testing stuff , trying out other stuff, etc... etc...) and Windows 10 threw a fit by giving me that well known BSOD style errors. 

I've got 2 PC's and 3 Laptops (though I've got enough spare parts as well... LOL! ). 

So now I've got the hard drive out of the laptop and performing some tests in my PC (as a precaution) then I've got to clean it up good as new. 

Once that's done, I just pop it back in the Laptop and do some magic installing the latest Windows 10 build fresh as new. 

I just love spares for Developments with Testing and such! I've even got a bunch of working Android for backups/developments of my App's and such. 

When i get Windows 10 going, I'm going to see what I can do. I'm just not a big fan of Windows 10 myself but, I need to know it to help out here. 

That's enough jabber/geek talk... LOL! 


~Ibuprophen

**Comment by ghosted17** [comment]
Been watching this for a while. Im getting back into dev and will take a look too. Hopefully we this fixed. I love dark window with classic, better then any other UI out there and this will look so good all dark. Anyway back to trying to fix my graphics card and wifi and auto mounts working under another OS which is well know for its bugs. At least win is has proper plug and play. that's enough geek talk from me 2 lol

**Comment by NomDeMorte** [comment]
Possibly related, using the Vista style navigation pane the icons and scrollbar also seem to be effected.

**Comment by blackcrack** [comment]
a hint.. more Programmers, more fixes.. do you know WinNT Programmers, ask they are, if they want help .. also for the later WinNT like Reactos (it will comes in any case... and this for free for all under GPL)
So ask around at known Programmers who you personally knows.. tell they are to be helping hand need here for patches and fixes..

best regards
blacky

**Comment by Ibuprophen** [comment]
I used to work with WinNT but, haven't really had the need to for a while. Maybe I should get it going again. 

Hmm... Maybe I can reparation the hard drive for a dual boot on my Windows 7 to add NT. Hmm...


~Ibuprophen

**Comment by blackcrack** [comment]
first boot harddisk win7
the 2nd harddisk with linux and bootloader..
this make i am loke so..

and have a 1TB harddisk as 3rd one who i store the things ..

and with bootmenu, like F12 or F9 or what ever can you call the boot menu out of the Bios, if's enabled ..

in Linux works Virtualbox also really pretty.. so, .. well.. all is possible..

**Comment by Ibuprophen** [comment]
I would just simplify it by Partitioning my 2TB Laptop HDD in half. The computer would see it as if it were 2 separate HDD's. This way each partition would have separate Windows 7 & NT in it. 

I would then create a dual boot menu so, when the Laptop boots up, there would be a menu to choose which one to boot into for 30 seconds to a minute (this is custom as it should be more than enough time to choose which one to select from). Of course, when the time period lapses, I would have a default OS pre-determined. 

It looks more complicated than it really is. It's a simple, and effective, way to do this. I used to do this when I used to dual boot Unix/Linux about a zillion years ago... LOL! 


~Ibuprophen

**Comment by blackcrack** [comment]
4 Primäre partitions, the first 2 for Win7 , one for WinNT, the 4th for Linux and Grub can maby managing all .. but if you reinstall kill Win7 your grub .. :\
so a bit heavy, but you could use grub4win https://sourceforge.net/projects/grub2win/..
oooorrr.. you use a usb-stick for boot, and install the linuxbootloader on the usb-stick, with them could you boot every your linux.. but you mus have the usb-stick ever at man..

well, the dualboot have i am also behind me ;) but a small ideen exchange is never bad :) :+1: 

your Blacky \*bg*

**Comment by Ibuprophen** [comment]
Ahhh... LOL! 

I'm kinda familiar with what you're describing. I do believe that a friend of mine has a similar/same setup on his Alternate PC. 

I have (legit ones of course) the Windows 7 Pro, 8.1 Pro, 10 Pro & NT on separate USB Thumb Drives.

Though I own pretty much all the Microsoft Software stemming back to MS-DOS 3.1. I even have all 25 Official (3.14") Floppies of Windows 95 too. 

For preparing the HDD partitions, I use the PartitionMagic Software and it's awesome too. I also use it because of the MicroSD's I prepare for the various Android devices I have. 

For Creating Updated (Windows OS) USB Thumb Drives, I primarily use the Rufus Software. That's a great piece of Software that I even helped out with (once in a blue moon) on a few occasions in Github. 

I tend to stick with the tools I use as long as it works out for me. It's like that saying "don't fix what's not broken". 

Before I had retired, some of the qualifications I had was a handful of Microsoft Certifications and I'm actually current as a registered Microsoft OEM System Builder too. 

The you go.... You pulled me into "jabbering mode"... LMAO! :-O 


~Ibuprophen

**Comment by individuell** [comment]
I kind of found a fix/workaround for the issue. If you right click the back and remove the checkmark it collapses to this small status bar which is more bearable.
![image](https://user-images.githubusercontent.com/5064362/64647850-1058bd00-d41a-11e9-912b-ad6e4d52ecc4.png)

**Comment by GermanKiwi** [comment]
> I kind of found a fix/workaround for the issue. If you right click the bar and remove the checkmark it collapses to this small status bar which is more bearable.

From your screenshot I deduce you're referring to the Quick Access Toolbar, which appears above the ribbon in the top-left corner of the Explorer window, and which is a Windows/Microsoft feature.

However, in this thread we're discussing the Classic Explorer status bar, which appears along the bottom edge of the Explorer window and has nothing to do with the Quick Access Toolbar - they serve different purposes.

**Comment by blackcrack** [comment]
@individuell , should maybe check which Windows version running and then set it up to remove the checkboxes, or so.. so include you're fix/work around... Do you be a programmer ?

@Ibuprophen  is this possible to add a Win NT-versions checker and add some scripts/code for the WinNT-Version (well, i am not a programmer, only a ooold 0X Dude) respective Windowsversion ? therewith could we service different WinNt versions.. winXP/Reactos, Win7, Win8 and 10 (in different versions, for bugfixes and workarounds)

... just so a idea .. *s*

best regards
Blacky

**Comment by Ibuprophen** [comment]
@blackcrack, TBH, it's been quite some time since I've used WinNT and, when I did last, I never used Classic Shell on it (Yes, I stated Classic Shell because it's been that long). 

Actually, I doubt that I would have really looked to use a Start Menu replacement software since, at the time, it had the Classic Start Menu already. 

Regarding compatibility for WinNT, I wasn't aware of/had any knowledge of any issues with WinNT. 

Do you know of any issues with (or does this software even support) WinNT @coddec, @ge0rdi & @XenHat? 

Regarding WinXP & WinVista, I haven't used them in quite some time either (not as long as WinNT). I believe that, ultimately, WinVista is actually a pseudo version of Windows 7 anyways. 

The Windows OS versions I typically have on my (test systems only) PC's/Laptop's are (the Professional versions of) Win7, Win 8.1 & Win10. 

My primary system is Win8.1 Pro that's used for a bunch of stuff that I "typically" don't test on since it has software like Android Studio and other biggies. It's just pain stakingly long to have to redo my Primary Tower PC. 


~Ibuprophen

**Comment by flatline4400** [comment]
Any update on this?  Failing something official, can we hack a resource somewhere to make the Classic Exporer Toolbar and the Status bar use a darker theme?

**Comment by Turk303** [comment]
![Whitebars - ability to set shade or colour would be so welcome](https://user-images.githubusercontent.com/52090402/76721973-b5da4980-6739-11ea-95a4-59a1b103a79e.PNG)
Am I in the right place to keep an eye on possible updates in the ability to convert the white bars in the image I've posted to either shades or colours? If so (FINALLY) I found "you" :)

**Comment by JohnEdwa** [comment]
Unless Open-Shell changes the way it creates these bars, there is very little chance of it fixing it without Microsoft doing it for us. Enabling Dark mode changes the normal Explorer status bar, all the context menus, details panel and, if enabled, the old-style Command Bar as well, but what isn't changed is the "Menu Bar", which is what Open-Shell uses, and the only way to change that currently is with a custom visual style.
![darkribbon](https://user-images.githubusercontent.com/5417183/78624129-85498380-7891-11ea-8723-aa94c3048989.PNG)

Though that system is also used for the Task Manager which currently stays blindingly white, so there is a small chance MS might make the change. It is after all buried deep in the OS for the "High-Contrast" theme, and can be accessed by custom styles too.

**Comment by Turk303** [comment]
Likely known by most here but just in case I thought I'd mention an app named QTTab does change the white bars (i believe from whatever shade is selected in Win 10's personalise control panel, but as "we" use Classical Shell's override to be able to use a darker/black theme (or maybe thats WinAero Tweaker I'm using as well... i think it might be regardless, alls fine with those 2, has been for 6+ months so check that out aswell - thats WINAERO TWEAKER, enables all kinds of tweaks n twiddles.) k back on track, how the Classic Shell and how QTTab interact fully I'm yet uncertain, seems ok though.

Just as classic shell is enabled/disabled, QTTab's options (enabled/disbaled & "theme's/actions) are also accessible via the same method of View/Options drop down in an open explorer window (as well its own opened app ui).

As said, all appears fine together with a glitch or 2 that is likely easily ironed out with the suitable set up choices but for now my icons are invisible due to QTTab making the white bar a black bar (as I want) but of course black icons are then invisible so I will be (when time allows) recreating the icons I've used so they are a light grey & thus my set up is usable (the image 2 above, which at the time of screen cap does have QTTab installed shows why I haven't used the 2 together for more than installation and basic tests.
One of the glitches I mentioned was QTTab either "removed" the white text, made it black (invisible) or resized something, I'm certain resizing of font or ? will fix that - this is why I'm posting, to see if anyone else is up for checking it out.

I'll return, provide some successful pics and results or? when time permits. 

Til then. L8rs

**Comment by flatline4400** [comment]
> Unless Open-Shell changes the way it creates these bars, there is very little chance of it fixing it without Microsoft doing it for us. Enabling Dark mode changes the normal Explorer status bar, all the context menus, details panel and, if enabled, the old-style Command Bar as well, but what isn't changed is the "Menu Bar", which is what Open-Shell uses, and the only way to change that currently is with a custom visual style.

The menu bar on my systems is black with white text while in dark mode (1909), regardless of classic shell being installed or not.

The color Open Shell picks for the background of the Classic Explorer Bar seems arbitrary, after a brief look at the code.  There's no reason why it couldn't pick it from a different Windows setting or, even better, a custom setting that's part of Open Shell itself.  It's just another custom toolbar, and they can look like whatever you want.  I'd try to fiddle with it myself but I've never tried to compile a big project like this before and I'm sure it would be hours before I got it to go, if ever.  The Status Bar may be a different beast, I haven't looked at that part.

**Comment by JohnEdwa** [comment]
>The menu bar on my systems is black with white text while in dark mode (1909), regardless of classic shell being installed or not.

The old "File Edit View Tool" menu bar that Classic Shell uses for its icons as well? 
Because if so, then that was fixed by Microsoft in 1909 - the image above is from 1803 where that menu follows the old "Classic" visual style colours that stay light even with dark mode. I can't test it because for me that menu bar is now actually black on black as I have QTtabBar taking over the skinning for it.

**Comment by Jevnaker** [comment]
Not sure if this is the right place to put this, but i recently upgraded my laptop to windows 10 from windows 7 and installed open shell ofc. But with the same settings in terms of skin and start menu style i get different colour on my laptop than i do on my desktop with the same settings in the options for openshell. also on my laptop only the clock part of my taskbar was changed to white colour when i changed the colour of it. The only difference is that my desktop uses an older version of openshell. How can i get the laptop version of my openshell to match my desktop one when i get different colours on the two with the same openshell skin settings.

**Comment by jabcreations** [comment]
Fix, kind of...

You'll need to get a copy of WindowBlinds (not free) and then use this free theme:
https://www.wincustomize.com/explore/windowblinds/8824

Unfortunately besides that WindowBlinds is NOT free this theme (the only one on the entire website since I last checked ~June 2020) this is the only theme to actually do an actual dark mode.

BUT it adds a dumb pointless button the simply collapses a window. It messes with the window controls for Waterfox making them useless.

There is an editor for *.WBA files and I have faintly found the reference to that damned fourth button. However I simply do not have the time to tinker until death.

Since I was kind enough to post this much maybe someone will be kind enough to dig in to the WBA editor, fix this theme and post the working file for everyone.

**Comment by itsdoreiku** [comment]
Still waiting on a real fix for this and not an extra 3rd party thing.

**Comment by Turk303** [comment]
> Not sure if this is the right place to put this, but i recently upgraded my laptop to windows 10 from windows 7 and installed open shell ofc. But with the same settings in terms of skin and start menu style i get different colour on my laptop than i do on my desktop with the same settings in the options for openshell. also on my laptop only the clock part of my taskbar was changed to white colour when i changed the colour of it. The only difference is that my desktop uses an older version of openshell. How can i get the laptop version of my openshell to match my desktop one when i get different colours on the two with the same openshell skin settings.

See my mention above re WINAERO

in short: you may simply have different colours set in the "Personalisation / Colours" settings - see if matching those helps 

As said above, winaero enables (allows you to select) actual BLACK should it be wanted, which MS disallows. [why? who knows].
 
They sacked half+ of their Win10 dev team a few years back (for some juicy inside info see you tube channel "Barnacules Nerdgasm" an ex miscrosoft team employee who despite having an exemplary record at microsoft, as many of his colleagues, "won" the chopped lottery when ms downsized despite windows 10 shall we say not being in a perfect ideal state when shoved out the door on us, for politeness sake.
He also dropped long before most knew the Win 7 was no longer being update after the end of this year - alas his channel, due to ill health, has less videos of late but there's enough already there of great interest.

**Comment by Turk303** [comment]
> Fix, kind of...
> 
> You'll need to get a copy of WindowBlinds (not free and it must always be running though not a terrible resource-hog) and then use this free theme:
> https://www.wincustomize.com/explore/windowblinds/8824
> 
> Unfortunately besides that WindowBlinds is NOT free this theme (the only one on the entire website since I last checked ~June 2020) this is the only theme to actually do an actual dark mode.
> 
> BUT it adds a dumb pointless button the simply collapses a window. It messes with the window controls for Waterfox making them useless.
> 
> There is an editor for *.WBA files and I have faintly found the reference to that damned fourth button. However I simply do not have the time to tinker until death.
> 
> Since I was kind enough to post this much maybe someone will be kind enough to dig in to the WBA editor, fix this theme and post the working file for everyone.
> 
> * Edit 1: this does not properly work with the Task Manager which is really obnoxious to me.
> * Edit 2: theming in Windows is a disaster since there is not much consistency. You can use WindowBlinds to edit themes though with a few hours I haven't had the success I've sought. Still, this should get most folks a bit further ahead.

I happen to have a *cough* "copy" of Windowsblinds and did semi succesfully tweak an existing theme but felt it may have introduced some mild instability (lets face it we dont need any more as windows 10 stands) & for the benefit of white to black seemed offset by the additional resources used.

We'll get there - i'm sure - maybe someone like nirsoft or sordum org will grow a clone of themselves and set them to task... they appear quite busy as is. :P

**Comment by ThumpieBunnyEve** [comment]
bump.

**Comment by jean3x7** [comment]
Any way to fix it? workaround?

**Comment by jabcreations** [comment]
You guys know developers accept money, right? Like, they have bills to pay and 99.999% of the people they talk with can't or won't give them money while saying, "gimme gimme gimme!" If I really wanted this fixed I'd offer the main developer $100+. Most code is atrocious, so working on code to deal with someone else's code takes ridiculous amounts of time. Developers have lives, family and friends too so the best thing someone could do is start some sort of tip-jar and allow people to add a few bucks that get deducted / paid to the developer when the problem is fixed. No one here might be able to drop $100 or more to fix this problem though there certainly are a lot of people who could chip in $5.

**Comment by blackcrack** [comment]
what do you think why Inkscape not really come in front, i have posted at they are a suggestion for autorun batch commands the switch yes for make default actions "-y" to have a possibility for let automatic png's move into svg's ,
one of them developer had take me away from the normal issue post and told me the idea is grate .. he want do into.. and behind a couple posts who have he make more hot to make integrate it into this f.. b.. switch told he to me, i have to make a payment for supporting this... and the issue was disappear in the normal issue list.. and something do i am not supporting.. look Gimp,
could be Gimp so far if they kidnaps users to say them, first a Champaign shower then the enhancement.. at this point raise i my middle finger ... we are here not on a shop.. a Donation page is really well for support the supporting, but explicit mulct/extort for supporting something.. oohh man.. not with me.. this was at inkscape.. so i raised the finger ..
And 100$ helps me over 2 weeks for eat.. And a Dontaion-page helps to give money and this Developer who makes more issues, becomes more from the money, this is the right way, but not a extort at any single users..
Therewith could you better advise if you consult a Donation page for Open-Shell to help the Developer for make his own money if they do something... and not push the blame to the users ...

best regards

**Comment by jabcreations** [comment]
This directory:
C:\Windows\Resources\Themes\aero

Contains *.msstyles files.

There are a TON of options and no doubt one of them can be edited.

Unfortunately the program said that Windows rejected the theme though perhaps someone with some patience may want to try and edit an existing Aero theme?

Here is the binary to edit *.msstyles files:
https://github.com/nptr/msstyleEditor/releases

**Comment by jedikalimero** [comment]
Still not a fix for this issue? 
Also, someone should change the topic to something like:
 "Classic Explorer status bar and toolbar don't change to dark mode in Windows 10"

**Comment by FlarosOverfield** [comment]
You know i wouldn't mind the fact that this issue is still present if it wasn't for the fact that YOU CANNOT DISABLE THE STATUS BAR.
**WHO THE F---- NEEDS TWO F------- STATUS BARS?????**

**Comment by blackcrack** [comment]
> You know i wouldn't mind the fact that this issue is still present if it wasn't for the fact that YOU CANNOT DISABLE THE STATUS BAR. **WHO THE F---- NEEDS TWO F------- STATUS BARS?????**

you know, who scream and have a behave  like an douchebag, have rarely right ...

**Comment by FlarosOverfield** [comment]
> > You know i wouldn't mind the fact that this issue is still present if it wasn't for the fact that YOU CANNOT DISABLE THE STATUS BAR. **WHO THE F---- NEEDS TWO F------- STATUS BARS?????**
> 
> you know, who scream and have a behave like an douchebag, have rarely right ...
> 
> and the best, any laugh about ;) because the moderator have the right, your posting to delete ;)

Explain the need for two status bars then. Without broken English, if possible.

**Comment by jabcreations** [comment]
It is possible to use either the Classic Explorer status bar or the built in Windows Explorer status bar.

1. Go to the Settings for Classic Explorer.
2. Click on "Status Bar" tab.
3. Check or uncheck "Show status bar".
4. Click "Ok".
5. Open a new instance of Windows Explorer.

I'm using WindowBlinds so I'm not sure if the remaining status bar will have an issue so you can alternatively use the built in status bar if you want:

1. Click the "View" menu in Windows Explorer.
2. Click on the "Options" button.
3. In the Folder Options modal click on the "View" tab.
4. Scroll about two-thirds the length of the scrollbar.
5. Check or uncheck "Show status bar".

**Comment by blackcrack** [comment]
Hi @jedikalimero  
> Still not a fix for this issue? Also, someone should change the topic to something like: "Classic Explorer status bar and toolbar don't change to dark mode in Windows 10"

i bet it's setted up a wrong darkmode color in "#color" in the CShell code, instead should use a variable of WinNT with (a.e.)"%Nightshapecolor%" who is political more correctly and use it directly from the System-"color"
come nuances can only arise if as shortshoot be use a hardcode color be use, because, if in the System change some color nuances arises then and this could be the result ..

if i be in able to presume that... in this case .. now is the question , exist some variable for the system ?
if yes, then why not use this variable .. and make directly the possible for change manually out an other color with a Textbox for css5(or higher) colors and add directly the support for actual css colors who is included transparency with the last 2 digits at 8 digits ( or how long is the actual css color digits ? 8 or more digits ? ) the actual html/css code support could be directly updated in OpenShell... imho

**Comment by EvocativeOlash** [comment]
> > > You know i wouldn't mind the fact that this issue is still present if it wasn't for the fact that YOU CANNOT DISABLE THE STATUS BAR. **WHO THE F---- NEEDS TWO F------- STATUS BARS?????**
> > 
> > 
> > you know, who scream and have a behave like an douchebag, have rarely right ...
> > and the best, any laugh about ;) because the moderator have the right, your posting to delete ;)
> 
> Explain the need for two status bars then. Without broken English, if possible.

Default status bar is horribly broken missing (free space), which is one of the major reason many of us installed this in the first place.  It was a built in feature in previous versions of windows removed for no discernable reason.  Their replacement of forcing you to navigate to and look at the drive tiles is just a waste of time.

**Comment by blackcrack** [comment]
@codec @ge0rdi

**Comment by ppvnf** [comment]
> It is possible to use either the Classic Explorer status bar or the built in Windows Explorer status bar.
> 
>     1. Go to the Settings for Classic Explorer.
> 
>     2. Click on "Status Bar" tab.
> 
>     3. Check or uncheck "Show status bar".
> 
>     4. Click "Ok".
> 
>     5. Open a new instance of Windows Explorer.

I'm confused. Do I need to download Classic Explorer instead of Open-Shell-Menu? I am not finding those options. Thanks. I just want to disable the useless white bar that Open-Shell-Menu created

**Comment by ldx00** [comment]
Shout out to @Turk303 above! I've now basically got at least something vaguely acceptable!

TL:DR - You can change the colour of the toolbar to a fixed colour using QTTabBar, but not the status bar. If you make it too dark then the menu items become unreadable.

Here's the useful post from @Turk303 !
https://github.com/Open-Shell/Open-Shell-Menu/issues/121#issuecomment-610178296

I installed QTTabBar from here
http://qttabbar.wikidot.com/

This almost got me what I wanted but the colour was off. Going into the QTTabBar Options > Appearance in the Nav bar on the left > then to the Toolbar tab on the top > then to Toolbar background color... allowed me to choose any colour I wanted for the QTTabBar and this was automatically applied to all the toolbars, including of course the Classic Explorer one!
![image](https://user-images.githubusercontent.com/87611793/230797129-429d65df-98d4-499c-9c9e-1d67f04f5924.png)

And here is how it looks!
![image](https://user-images.githubusercontent.com/87611793/230797361-a8393f6c-66b8-4b67-8d9a-4532f5afbf7c.png)

Issues (some quite big possibly...)
- You can't change the font colour of the menu text, so it's very hard to read the menu items on the left, but I know what they are, so I don't really care
- The status bar is sadly not fixed
- You need to keep at least one of the toolbars enabled, otherwise the toolbar reverts to white again. Thankfully there is basically an invisible one (QT Base Toolbar) that you can see in the second screenshot above, that if you enable, will keep the toolbar colour from reverting to white
- There are lots of random options and explorer enhancements that QTTabBar adds, you need to go through the options and disable what you don't want. There does actually seem to be some good stuff there though that some might find useful
- Sometimes the menu items on the left are gone altogether and all my icons are on the left. I did see this occasionally in the past, but during my playing around it happened a lot more often. I'm hoping that after a restart it'll stabilise, let's see!
- This colour is fixed so if you switch between light and dark mode, maybe by time of day for example, then you will have to keep going in and changing the colour manually. I stick with dark mode permanently so it's not an issue for me.

For now at least this workaround is good enough for me. If anyone manages to find out how to change the font colour, please let me know, then it'll be almost perfect (apart from the fact this isn't a proper fix).

A couple more points
- I thought I might be able to find some kind of registry edit when I changed the colour, but no such luck, at least nothing showed up with regshot that is very useful for doing a before and after comparison on your registry
https://github.com/Seabreg/Regshot
- I think from my digging that the colour of the toolbar is coming from a legacy theme setting somewhere, as people seem to have had some luck when they find a custom theme that uses a darker colour for "something". I say something here because it may not be the toolbar colour, just some colour in the theme and standard themes seem to use white here. The fact that someone else, when upgrading a few PCs managed to get different colours there seems to indicate this is not hardcoded somewhere, but is being picked up from the colour of something else. I suspect that the newer OSs like 10 and 11 don't specify that colour any more, and dark mode colours are again defined differently so there is no native way in these OSs to modify that colour. Good luck to the devs in finding this!

Big thanks to @Turk303 again for his useful post! I'd almost given up!

Other related bugs
#335 #361 #681 #772 #827

**Comment by pbakes** [comment]
> Can confirm this is a issue. Dark mode selected in Windows 10 and the Classic Explorer bars don't match Dark Mode.
> 
> EDIT: ![image](https://user-images.githubusercontent.com/27028529/58226444-0856ff00-7cfd-11e9-90f2-e1a381ea8bdd.png)

Has anyone found a fix for this?

**Comment by pbakes** [comment]
> 

Nevermind, this is the answer
